src/frinkiac.coffee

```diff
@@ -80,7 +80,7 @@ combineCaptions = (captions) ->
     captions[0].Content
 
 module.exports = (robot) ->
-  robot.respond /(simpsons search|frinkiac) (.*)/i, (msg) ->
+  robot.respond /(simpsons search|frinkiac|me) (.*)/i, (msg) ->
```

Comment:
Hmmm, I believe this would match `hubot me`. Perhaps `/(simpsons (search|me)|frinkiac) (.*)/` would give you what you want?